### PR TITLE
Model section with list and form

### PR DESCRIPTION
**Pull request by edoardocavazza**
Waiting for https://github.com/bedita/bedita-sdk-js/pull/10

List, create and edit `ObjectTypes` with custom properties.

### CODE REVIEW

**Comment by edoardocavazza** [comment]
First preview https://deploy-preview-79--be4-manager.netlify.com

File: src/components/be-model-form/be-model-form.js [hunk 1]
```diff
@@ -22,20 +22,73 @@ export class BEModelForm extends mix(Component).with(ModelMixin, FormatterMixin,
...
+                        .then((properties) => {
+                            this.dynamicProperties = properties;
+                        });
+
+                    this.factory('model').getPropertyTypes()
+                        .then((types) => {
+                            this.types = types;
+                        });
+                    
+                    this.factory('model').getAbastractObjectTypes()
```
Comment (didoda) [inline src/components/be-model-form/be-model-form.js]: `typo` o è proprio `getAbastractObjectTypes`?

File: src/mixins/model-mixin.js [hunk 2]
```diff
@@ -188,3 +174,12 @@ function setSelectValues(select, values) {
         }
     );
 }
+
+/**
+ * Extract associated model
+ * @param {Element|BaseComponent} target 
+ */
```
Comment (didoda) [inline src/mixins/model-mixin.js]: manca `@return` in jsdoc ;)

**Comment by stefanorosanelli** [comment]
Good job @edoardocavazza!

Some improving suggestions, can be done after merge:
 * `object types` card display - ok - `object types` can be enabled/disabled setting `enabled` true or false - we should view enabled and disabled types differently highlighted and we should be able to enable/disable a type
* `enable`, `parent_name` and `table` attributes should be displayed in `type` detail
* there is a problem in object types `save`, but it's and API bug
* all properties (dynamic & static) are available with `/model/properties?filter[object_type]={type}` we could safely use this one call and then differentiate between static and dyncamic checking id's: numeric are dynamic, `uuid like` are static - doing so we don't miss properties, currently, using parents properties we miss some (see `profiles` or `users` for instance)
* for every property then:
  - if `static` we can only hide id --> simply add to `hidden` object type property
 - if `dynamic` we can change or remove it
 - we can add new dynamic properties

**Comment by stefanorosanelli** [comment]
Static and dynamic properties groups are OK!

* somehow `ADD` button for a new dynamic property is not working
* Travis CI and Netlify deploy both fail with same error:
```
bundling... (/home/travis/build/bedita/manager/src/main.js)
{ Error: 'default' is not exported by node_modules/@chialab/synapse/src/vendors/pouchdb.js
    at error (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:170:15)
    at Module.error (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:17023:9)
    at Module.trace (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:17151:22)
    at ModuleScope.findVariable (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:14295:28)
    at BlockScope.Scope.findVariable (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:11443:69)
    at Node.Identifier.bindNode (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:12007:40)
    at NodeBase.bind (/home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:11021:14)
    at /home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:11027:56
    at /home/travis/.config/yarn/global/node_modules/rollup/dist/rollup.js:11044:17
    at Array.forEach (native)
  code: 'MISSING_EXPORT',
  url: 'https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module',
  pos: 537,
  loc: 
   { file: '/home/travis/build/bedita/manager/node_modules/@chialab/synapse/src/helpers/db.js',
     line: 2,
     column: 7 },
  frame: '1: import { Emitter } from \'@chialab/proteins/src/factory.js\';\n2: import PouchDB from \'../vendors/pouchdb.js\';\n          ^\n3: import { internal } from \'./internal.js\';\n4: import { DBOpeningErrorException } from \'../exceptions/db-opening-error.js\';' }
error bundling Main
```

**Pull request merged**